xml/ha_yast_cluster.xml

```diff
@@ -822,186 +1165,60 @@
       <imagedata fileref="yast_cluster_services.png" width="75%"/>
      </imageobject>
      <textobject role="description">
-       <phrase>
-         The Service screen lets you start or stop services. You can enable or disable the
-         cluster starting at boot time, start or stop &pace; and &corosync;, and start or stop
-         &qdevice;. You can also open the required firewall port.
-       </phrase>
-     </textobject>
+      <phrase>
+        The Service screen lets you start or stop services. You can enable or disable the
+        cluster starting at boot time, start or stop &pace; and &corosync;, and start or stop
+        &qdevice;. You can also open the required firewall port.
+      </phrase>
+    </textobject>
     </mediaobject>
    </figure>
   </sect1>
 
-  <sect1 xml:id="sec-ha-installation-setup-csync2">
+  <sect1 xml:id="sec-ha-setup-yast-csync2-sync">
    <title>Transferring the configuration to all nodes</title>
    <para>
-    Instead of copying the resulting configuration files to all nodes
-    manually, use the <command>csync2</command> tool for replication across
-    all nodes in the cluster.
-   </para>
-   <para>
-    This requires the following basic steps:
+    After the cluster configuration with &yast; is complete, use <command>csync2</command>
+    to copy the configuration files to the rest of the cluster nodes. To receive the files,
+    nodes must be included in the <guimenu>Sync Host</guimenu> group you configured in
+    <xref linkend="pro-ha-installation-setup-csync2-yast"/>.
    </para>
+   <para> Before running &csync; for the first time, you need to make the
+    following preparations: </para>
```

Comment:
Worth to share another simplest one is `crm cluster init -y csync2` which basically implement "PROCEDURE 7.10"

Reply:
Ohh, thanks, that's much easier!